README.rst

```diff
@@ -135,3 +135,11 @@ It's unnecessary to use a ``list`` around list comprehension, since it is
 equivalent without it. For example:
 
 * Rewrite ``list([f(x) for x in foo])`` as ``[f(x) for x in foo]``
+
+C412: Unnecessary list comprehension - rhs of 'in' can be a generator
```

Comment:
Hey, a couple things:

1. "rhs" isn't obviously "right hand side" to every developer. I think renaming the rule to "Unnecessary list comprehension - 'in' can take a generator" is better wording. It's clear which side of `in` should be the generator. Similarly can you reword the below paragraph?
2. Can you also add a line to the table at the top of the doc with the rule, around line 54?

Reply:
Thanks. Updated in 87781b1.